rskj-core/src/test/java/org/ethereum/config/blockchain/upgrades/ActivationConfigsForTest.java

```diff
@@ -177,7 +177,8 @@ private static List<ConsensusRule> getArrowhead631Rskips() {
     private static List<ConsensusRule> getLovell700Rskips() {
         return new ArrayList<>(Arrays.asList(
             ConsensusRule.RSKIP427,
-            ConsensusRule.RSKIP428
+            ConsensusRule.RSKIP428,
+            ConsensusRule.RSKIP454
```

Comment:
unrelated to this pr but arent we missing rskip438 here?

Reply:
Yes, we are. I will add it if we ok on doing it in this PR